RELEASE-NOTES.md

```diff
@@ -27,6 +27,11 @@
 - Fixed `sample_ppc` and `sample_ppc_w` to iterate all chains(#2633, #2748)
 - Add Bayesian R2 score (for GLMs) `stats.r2_score` (#2696) and test (#2729).
 - SMC works with transformed variables (#2755)
+- Speedup OPVI (#2759)
```

Comment:
I think you should add a new PR for this, and put the below Deprecations into a new unreleased (making the deprecation the next release)

Reply:
But we have not yet released 3.3, only release candidate

Reply:
If its already been deprecated, we can remove it in 3.3